cssom-view/Overview.bs

```diff
@@ -1301,7 +1298,9 @@ The <dfn attribute for=Element>clientHeight</dfn> attribute must run these steps
 <h3 id=element-scrolling-members>{{Element}} Scrolling Members</h3>
 
 To <dfn>scroll an element into view</dfn> <var>element</var>,
-with a {{ScrollIntoViewOptions}} dictionary <var>options</var>,
+with a scroll behavior <var>behavior</var>,
+a block flow direction position <var>block</var>,
```

Comment:
and an optional block flow ... and an inline base ...

Reply:
Not sure I follow. Would you like me to tweak the wording? Or make these optional? Both?

Reply:
I think you treat the second and third argument as optional because there's one invocation that just invokes this with "auto", afaict.

Reply:
I don't see that. Within cssom-view it's only invoked here
```
1. <a lt='scroll an element into view'>Scroll the element into view</a>
    with <var>behavior</var>, <var>block</var>, and <var>inline</var>.
```

Reply:
Yeah you're right. I got confused with "scroll an element".